monitor/views.py

```diff
@@ -104,6 +105,21 @@ def queries_stats_ajax(request):
     return JsonResponse(json.load(urllib2.urlopen(req)))
 
 
+@login_required
+def api_usage_stats_ajax(request):
+    query = urllib.urlencode({
+        'query': 'api_client_username:%s' % (request.user.username),
+        'range': '604800',
+        'filter': 'streams:530f2ec5e4b0f124869546d0',
```

Comment:
magic number - if something goes wrong with our graylog instance and we have to reinstall it, this will probably change